web/src/main/java/se/inera/intyg/rehabstod/web/controller/api/dto/SickLeavesFilterRequestDTO.java

```diff
@@ -21,22 +21,25 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import se.inera.intyg.rehabstod.service.diagnos.dto.DiagnosKapitel;
 
+@Data
+@NoArgsConstructor
 public class SickLeavesFilterRequestDTO {
     private List<String> doctorIds;
-    private int toSickLeaveLength;
-    private int fromSickLeaveLength;
+    private Integer toSickLeaveLength;
+    private Integer fromSickLeaveLength;
+    private List<DiagnosKapitel> diagnosisChapters;
 
-    public SickLeavesFilterRequestDTO() { }
-
-    public SickLeavesFilterRequestDTO(List<String> doctorIds, int toSickLeaveLength, int fromSickLeaveLength) {
+    public SickLeavesFilterRequestDTO(
+        List<String> doctorIds, int toSickLeaveLength, int fromSickLeaveLength, List<DiagnosKapitel> diagnosisChapters
```

Comment:
Since we decided to use @NoArgsConstructor, maybe we can use @AllArgsConstructor to generate this constructor?